homeassistant/components/whirlpool/quality_scale.yaml

```diff
@@ -0,0 +1,81 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      The integration does not provide any additional actions.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup: done
+  entity-unique-id: done
```

Comment:
Can the username change?

Reply:
I am not sure I understand. The username is not used for the entity unique id.

Reply:
Oh it is for the config entry one, wrong rule

Reply:
ah! no, AFAIK it cannot be changed.